src/main/groovy/edu/oregonstate/mist/webapiskeleton/SkeletonApplication.groovy

```diff
@@ -26,7 +29,11 @@ class SkeletonApplication extends Application<Configuration> {
      * @param bootstrap
      */
     @Override
-    public void initialize(Bootstrap<Configuration> bootstrap) {}
+    public void initialize(Bootstrap<Configuration> bootstrap) {
+        bootstrap.addBundle(new TemplateConfigBundle(
+                new TemplateConfigBundleConfiguration().charset(Charsets.US_ASCII)
```

Comment:
Not Charsets.UTF_8? Actually, it looks like that's the default if you don't pass a TemplateConfigBundleConfiguration.